integration_tests/tests/integration_tests/snapshots/integration_tests__token_extensions_tests__token_extensions_get_asset.snap

```diff
@@ -0,0 +1,63 @@
+---
+source: integration_tests/tests/integration_tests/token_extensions_tests.rs
+expression: response
+snapshot_kind: text
+---
+{
+  "interface": "FungibleToken",
```

Comment:
If the ownership model is single should this be classified as an NFT?

Reply:
changing ownership to token when token is first created